src/VisualStudio/Core/Def/Implementation/Diagnostics/VisualStudioDiagnosticAnalyzerProvider.cs

```diff
@@ -138,41 +124,10 @@ internal static ImmutableArray<HostDiagnosticAnalyzerPackage> GetHostAnalyzerPac
                 //
                 // fortunately, this only happens on disposing at shutdown, so we just catch the exception and silently swallow it. 
                 // we are about to shutdown anyway.
-                return ImmutableArray<HostDiagnosticAnalyzerPackage>.Empty;
+                return ImmutableArray<AnalyzerReference>.Empty;
             }
         }
 
-        private static void EnsureMandatoryAnalyzers(ImmutableArray<HostDiagnosticAnalyzerPackage> packages)
-        {
-            foreach (var package in packages)
-            {
-                if (package.Assemblies.Any(a => a?.EndsWith(MicrosoftCodeAnalysisCSharp, StringComparison.OrdinalIgnoreCase) == true) &&
-                    package.Assemblies.Any(a => a?.EndsWith(MicrosoftCodeAnalysisVisualBasic, StringComparison.OrdinalIgnoreCase) == true))
-                {
-                    return;
-                }
-            }
-
-            FailFast.OnFatalException(new Exception("Mandatory analyzers are missing"));
```

Comment:
Did this move somewhere else or just figuring it's not really worth it? (It does seem of questionable value...)

Reply:
Removed. It used to be necessary since we had some assumptions elsewhere. Now we don't. We can have solution without C#/VB compiler analyzers and works just fine.